src/test/ui/less_than_greater_than.stderr

```diff
@@ -0,0 +1,7 @@
+error: invalid comparison operator `<>`
+  --> $DIR/less_than_greater_than.rs:2:22
+   |
+LL |     println!("{}", 1 <> 2);
+   |                      ^^^ help: `<>` is not a valid comparison operator, use `!=`
```

Comment:
the span is too wide

Reply:
Good job on spotting that! Now I just have to figure out how to fix it.